solutions/gold/sumdiv.mdx

```diff
@@ -0,0 +1,80 @@
+---
+id: sumdiv
+source: Kilonova
+title: Sum of divisors
+author: Mihnea Brebenel
+---
+
+## Explanation
+
+The sum of all divisors of the integer $N$ can be efficiently computed in $\mathcal{O}(\sqrt{N})$ using the [divisor function](https://en.wikipedia.org/wiki/Divisor_function).
+The complexity comes from the prime factorization of $N$.
```

Comment:
what does this mean

Reply:
whats wrong with it

Reply:
It means that you can compute the sum of divisors from the prime factorization, which is faster than $O(N)$ brute force approach traversing the interval $[1, N/2]$

Reply:
![image](https://github.com/cpinitiative/usaco-guide/assets/59292725/c8edb4bb-bf34-4091-9175-2d43c7eafbff)

It says that on the same page :))

Reply:
![image](https://github.com/cpinitiative/usaco-guide/assets/59292725/a71fc5d8-da66-4144-94e5-60a8bca4767e)

Reply:
It says right there :))

Reply:
Case closed

Reply:
https://usaco.guide/gold/divisibility

Is on the same page the problem is

Reply:
i think the problem should be moved to modular arithmetic